### DISCUSSION TITLE
Add MPH as an option for Telemetry View

### DISCUSSION THREAD

**Discussion by TheBaconCat** [f1multiviewer/issue-tracker#59] [Feature requests] [answered]
Would be a great option to add a setting to change from KPH to MPH based on user preference when viewing telemetry and other data. Extending this further there could be an option to switch from imperial to metric measurements! Thanks for the great app!!!

**Comment by f1multiviewer** [comment, marked as answer]
This is available now!

**Reply by TheBaconCat** [reply]
Awesome Thank You! Just in Time for the Race Weekend!
